tests/helper.ts

```
@@ -38,11 +96,11 @@ export type TestedGeojson = {
 export const countriesForWorkflow = getJSON({
   fileName: "countries-for-workflow",
   fileFolder: "lookup-data",
-}) as string[];
+});
 
 export const countriesToTestArray = !!process.env
   .IS_TESTING_BUSINESS_COUNTRIES_IN_A_ROW_AT_INSIGHTS_API
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)